client/src/pages/SignupOne.js

```diff
@@ -0,0 +1,225 @@
+/** @format */
+
+import React, { useState, useEffect } from "react";
+import { Box, Container, Grid, Typography } from "@material-ui/core";
+import { TextField, Button } from "@material-ui/core";
+import { Link } from "react-router-dom";
+import { useStyles } from "../themes/theme";
+
+const SignupOne = ({ setData }) => {
+	const classes = useStyles();
+
+	const [state, setState] = useState({
+		name: "",
+		emailSignup: "",
+		pwdSignup: "",
+		confirmPwdSignup: "",
+		userInfo: {},
+		nameError: false,
+		emailError: false,
+		pwdError: false,
+		confirmPwdError: false,
+	});
```

Comment:
Please see my comments for the `Signin.js` file. Not only for this particular segment and for the segments below, in this file, that involve not using camel case notation and using extra ternary operators.